addOns/webuipoc/src/main/pocs/reactWebUI/package.json

```diff
@@ -43,6 +43,9 @@
     "eslint-plugin-react": "^7.34.1",
     "eslint-plugin-react-hooks": "^4.6.2",
     "nth-check": ">=2.0.1",
+    "prettier": "3.3.3",
```

Comment:
How is it being executed?

Reply:
fixed

Reply:
Added a script in package.json

Reply:
This needs to be wired into the build, CI should check and fail the build if not properly formatted.